src/sdl_texture_mjpg.cpp

```diff
@@ -14,12 +15,20 @@ SDLTextureMJPG::~SDLTextureMJPG() {
   free(rgbdata_);
 }
 
-void SDLTextureMJPG::decompress(const unsigned char* jpeg,
-                                unsigned long jpeg_size) {
+int SDLTextureMJPG::decompress(const unsigned char* jpeg,
+                               unsigned long jpeg_size) {
   struct jpeg_error_mgr jerr;
   struct jpeg_decompress_struct cinfo;
   cinfo.err = jpeg_std_error(&jerr);
 
+  JpegErrorManager jpegErrorManager(cinfo);
+  if (setjmp(jpegErrorManager.escape_)) {
+    /* libjpeg found an error */
+    jpeg_destroy_decompress(&cinfo);
+    printf("JPEG decompression error");
```

Comment:
This should be EPRINT, we don't use printf like this is twincam, because we log errors to syslog and such